modules/secrets-store-azure.adoc

```diff
@@ -0,0 +1,308 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets-store.adoc
+
+:_content-type: PROCEDURE
+[id="secrets-store-azure_{context}"]
+= Mounting secrets from Azure Key Vault
+
+You can use the {secrets-store-operator} to mount secrets from Azure Key Vault to a CSI volume in {product-title}. To mount secrets from Azure Key Vault, your cluster must be installed on Microsoft Azure.
+
+.Prerequisites
+
+* Your cluster is installed on Azure.
+* You have installed the {secrets-store-operator}. See _Installing the {secrets-store-driver}_ for instructions.
```

Comment:
```suggestion
* You have installed the {secrets-store-operator}. See "Installing the {secrets-store-driver}" for instructions.
```
See previous comment.

Reply:
Same answer as other comment; leaving for now